deploy/config/local/server.conf

```diff
@@ -138,6 +138,11 @@ server {
         deny all;
     }
 
+    # Deny access to /wp/wp-cron.php /wp/xmlrpc.php, /wp/wp-admin/install.php and /wp/wp-admin/upgrade.php
```

Comment:
Hi @EarthlingDavey thanks 😄 
Is this direct access only? 
How does the CRON work on Justice UK?

Reply:
Thanks for the review :)
This will deny any access for these files via nginx.
Because there aren't any heavy scheduled tasks on Justice UK there wasn't a need to turn off WP_CRON like with the intranet.